recipe/conda_build_config.yaml

```diff
@@ -12,3 +12,7 @@ libclang_soversion:
 # keep for use with root when it catches up
 variant:
   - default
+
+with_cfg:
+  - true
+  - false
```

Comment:
Do we want to restrict this to unix? 🤔 

We're currently producing two separate builds on windows, though they end up being identical.